misk-gcp-testing/src/main/kotlin/misk/cloud/gcp/spanner/GoogleSpannerEmulator.kt

```diff
@@ -92,8 +92,9 @@ class GoogleSpannerEmulator @Inject constructor(
       .build()
     val docker: DockerClient =
       DockerClientImpl.getInstance(defaultDockerClientConfig, httpClient)
-    const val IMAGE = "gcr.io/cloud-spanner-emulator/emulator"
+    const val IMAGE_NAME = "gcr.io/cloud-spanner-emulator/emulator"
     const val CONTAINER_NAME = "misk-spanner-testing"
+    var image: String = "$IMAGE_NAME:latest"
```

Comment:
This variable is making the hairs stand up on the back of my head because it creates some unintuitive side effects. Examples:

```
// no pull
doStart() // starts latest

pullImage() // pulls latest
doStart() // starts latest

pullImage("1.0.0") // pulls 1.0.0
doStart() // starts 1.0.0
```

This is sort of exacerbated by the fact that docker doesn't require images to be pulled before start (if the image isn't present at start, it will pull automatically).

What do you think about passing `imageVersion` into `doStart(...)` as well? That way it's more clear that the command starts the specific version.

Reply:
oh ya that is kinda odd, I'll update